app/components/common/SystemCell.stories.js

```diff
@@ -0,0 +1,33 @@
+import { storiesOf } from '@storybook/react-native'
+import { action } from '@storybook/addon-actions'
+import React from 'react'
+import { scale } from 'react-native-size-matters'
+import Icon from 'react-native-vector-icons/Ionicons'
+import colors from '../../config/colors'
+import SystemCell from './SystemCell'
+const image = '../../assets/images/pallet-racking.png'
+const icon = (
+  <Icon name="hammer-outline" size={scale(20)} color={colors.white} />
+)
+
+storiesOf('System Cell', module)
+  .add('with image', () => <SystemCell imageUrl={require(image)} />)
+  .add('with image and title', () => (
+    <SystemCell
+      icon={icon}
+      imageUrl={require(image)}
+      title={'Pallet Racking'}
+    />
+  ))
+  .add('with image, title and no icon', () => (
+    <SystemCell imageUrl={require(image)} title={'Pallet Racking'} />
+  ))
+  .add('without image', () => <SystemCell title={'Pallet Racking'} />)
+  .add('with onPress', () => (
```

Comment:
With the storybook stories I would be a bit more explicit in which each story represents.

For instance I would change:
Line 15 to 'with image and title' => 'with image, title and icon' 
Line 25 'without image' => 'with title'
Line 26 'with onPress' => 'with icon, image, title and onPress'

You want this to act like documentation so make it as explicit as possible for any level of developer.